web-server/src/utils/Errors.ts

```diff
@@ -0,0 +1,20 @@
+type Error = {
+  status?: any;
+  raw?: any;
+}
+
+export type UserError = Error & {
+  id?: string;
+  email?: string;
+  username?: string;
+  password?: string;
+  passwordConfirm?: string;
+  country?: string;
+  show?: string;
```

Comment:
It's probably a good idea to remove this as well. You only get this from the /show/ endpoint and this is the only thing there. In fact, this string doesn't actually mean anything since any non-empty value means "User not found". Just return an empty 400 from that endpoint.

Reply:
401 error means "You're not logged in"